aloe_webdriver/screenshot_failed.py

```diff
@@ -45,9 +45,30 @@
 # pylint:disable=no-member
 
 
+if 'SCREENSHOT_DIR' not in globals():
+    SCREENSHOT_DIR = None
+
+
 FORMAT = 'failed_{feature_file}_{scenario_index}_{scenario_name}{outline_index}'
 
 
+def set_save_directory(directory):
```

Comment:
It is possible to set `aloe_webdriver.screenshot_failed.FORMAT` (and `SCREENSHOT_DIR` with your changes) directly. Why the extra functions?